Topic5Mysql/MySQLFiles/high-school-DB.sql

```diff
@@ -0,0 +1,169 @@
+CREATE DATABASE  IF NOT EXISTS `high_school` /*!40100 DEFAULT CHARACTER SET utf8 */;
+USE `high_school`;
+-- MySQL dump 10.13  Distrib 5.6.17, for Win32 (x86)
+--
+-- Host: localhost    Database: high_school
+-- ------------------------------------------------------
+-- Server version	5.6.23-log
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Table structure for table `course`
+--
+
+DROP TABLE IF EXISTS `course`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `course` (
+  `course_id` int(11) NOT NULL AUTO_INCREMENT,
+  `course_name` varchar(45) NOT NULL,
+  `hours_by_week` int(10) unsigned NOT NULL,
+  `schedule_time` int(11) NOT NULL,
+  `teacher_assigned` int(11) NOT NULL,
+  PRIMARY KEY (`course_id`),
+  KEY `teacher_id_idx` (`teacher_assigned`),
+  KEY `schedule_id_idx` (`schedule_time`),
+  CONSTRAINT `schedule_id` FOREIGN KEY (`schedule_time`) REFERENCES `schedule` (`schedule_id`) ON DELETE NO ACTION ON UPDATE NO ACTION,
+  CONSTRAINT `teacher_id` FOREIGN KEY (`teacher_assigned`) REFERENCES `teacher` (`teacher_id`) ON DELETE NO ACTION ON UPDATE NO ACTION
+) ENGINE=InnoDB AUTO_INCREMENT=6 DEFAULT CHARSET=utf8;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `course`
+--
+
+LOCK TABLES `course` WRITE;
+/*!40000 ALTER TABLE `course` DISABLE KEYS */;
+INSERT INTO `course` VALUES (1,'Estructuras de Datos',12,1,3),(2,'Bases de Datos',8,2,2),(3,'Computacion Grafica',10,3,1),(4,'Organizacion de Computadoras',11,4,2),(5,'Seguridad en Sistemas',4,5,1);
+/*!40000 ALTER TABLE `course` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `notes`
+--
+
+DROP TABLE IF EXISTS `notes`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `notes` (
+  `student_id` int(11) NOT NULL,
+  `course_id` int(11) NOT NULL AUTO_INCREMENT,

```

Comment:
This value is a foreign key, it shouldn't be autoincrement, since it is related to the table course.